src/dlgTriggerEditor.cpp

```diff
@@ -504,6 +504,12 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     showDebugAreaAction->setStatusTip(tr("Shows/Hides the separate Central Debug Console - when being displayed the system will be slower."));
     connect(showDebugAreaAction, &QAction::triggered, this, &dlgTriggerEditor::slot_debug_mode);
 
+    QShortcut *nextSectionShortcut = new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Tab), this);
```

Comment:
You can use `auto` here not to repeat the type

Reply:
I've got a habit of mimicking whatever coding exists around what I'm putting in place, which is probably why I didn't use auto. I think I still think about auto as being a bad thing, which it isn't. Gotta retrain those instincts! I'm about to do a commit to address this.